src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -552,6 +564,37 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                 }
             }
         }
+        public Dictionary<string, string> GetSsoPolicyHeaders()
+        {
+            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+            Debug.Assert(s_lazyCore.Value != null, "Should not call this API if MSAL runtime init failed");
```

Comment:
>Should not call this API if MSAL runtime init failed [](http://example.com/codeflow?start=52&length=52)

I don't believe this is a requirement. 
And we should exit out if we know we cannot call the policy api here.

Reply:
This only executes in DEBUG mode (so it's removed by compiler in the final bits). It's a message for the MSAL team dev, not for app deveveloper.